bundles/org.openhab.core.automation.module.script.rulesupport/src/main/java/org/openhab/core/automation/module/script/rulesupport/loader/AbstractScriptFileWatcher.java

```diff
@@ -67,7 +67,7 @@
  * @author Jan N. Klug - Refactored dependency tracking to script engine factories
  */
 @NonNullByDefault
-public abstract class AbstractScriptFileWatcher extends AbstractWatchService implements ReadyService.ReadyTracker,
+public class AbstractScriptFileWatcher implements WatchService.WatchEventListener, ReadyService.ReadyTracker,
```

Comment:
If it is no longer an abstract class the name `AbstractScriptFileWatcher` is wrong.